### ISSUE TITLE
Multiple Accessibility failures due to label placement

### ISSUE THREAD

**Issue by zgrossbart** [IBM/carbon-components-react#1163]
This is very similar to issue #1131.

When using a multiple components, the <label> element is being rendered after the <input> element, causing the following accessibility error: `Text inputs and select elements must have a label before the input control.`

This happens with the following elements:

See `TextInput` here: http://react.carbondesignsystem.com/?selectedKind=TextInput&selectedStory=enabled&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

See `TextArea` here:  http://react.carbondesignsystem.com/?selectedKind=TextArea&selectedStory=enabled&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

See `DatePicker` here:  http://react.carbondesignsystem.com/?selectedKind=DatePicker&selectedStory=single%20with%20calendar&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

See `TimePicker` here: http://react.carbondesignsystem.com/?selectedKind=TimePicker&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by zgrossbart** [comment]
This is a duplicate of #1149.  I'm closing this issue.
